Ujaan/Lab-2/Part-3/flake.nix

```diff
@@ -0,0 +1,36 @@
+{
+  description = "part 2 flake";
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-23.05";
+    utils.url = "github:numtide/flake-utils";
+    hello_lib.url = "github:KingSyclone/nix-hello-lib";
+  };
+
+  outputs = { self, nixpkgs, utils, hello_lib }:
+  let
+    pkgs = import nixpkgs {
+      system = "x86_64-linux";
+    };
+
+   in {
+    packages.x86_64-linux.default = pkgs.callPackage ./default.nix {
+      hello_lib = hello_lib.packages."x86_64-linux".default;
+    };
+
```

Comment:
instead of doing this, please use the overlay that you define in your upstream repo to get your library